src/clients/beacon-client/BeaconClient.ts

```diff
@@ -50,7 +50,7 @@ export abstract class BeaconClient {
     this.initSDK().catch(console.error)
   }
 
-  public async resetSDK(): Promise<void> {
+  public async destroy(): Promise<void> {
```

Comment:
@AndreasGassmann please add the documentation for this method explaining when to use it and the consequences (client instance cannot be used anymore).